CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ All notable changes to this project will be documented in this file.
 * feat: warn when hooked module is already loaded [#2926](https://github.com/open-telemetry/opentelemetry-js/pull/2926) @nozik
 * feat: implement OSDetector [#2927](https://github.com/open-telemetry/opentelemetry-js/pull/2927) @rauno56
 * feat: implement HostDetector [#2921](https://github.com/open-telemetry/opentelemetry-js/pull/2921) @rauno56
+* refactor(metrics-sdk): rename InstrumentationLibrary -> InstrumentationScope [#2959](https://github.com/open-telemetry/opentelemetry-js/pull/2959) @pichlermarc
```

Comment:
There are two CHANGELOG files: one in the root directory and one in the `exprimental/`. I find that people are very easy to overlook the one in the `experimental/` directory. Maybe we should merge those two.

Reply:
in this case you should modify both changelog files like you did.

> There are two CHANGELOG files: one in the root directory and one in the `exprimental/`. I find that people tend to overlook the one in the `experimental/` directory. Maybe we should merge those two.

I thought about this, but I think it makes for a confusing changelog file.

Reply:
```suggestion
* `@opentelemetry/core`: add InstrumentationScope [#2959](https://github.com/open-telemetry/opentelemetry-js/pull/2959) @pichlermarc
```

Reply:
Changed the changelog entry for stable in [b5296d9](https://github.com/open-telemetry/opentelemetry-js/pull/2959/commits/b5296d91c6d4c1a0566f40c96f3a2a359d72307c) :slightly_smiling_face: